packages/app/src/cli/services/context.ts

```diff
@@ -151,7 +169,9 @@ export async function ensureDevContext(options: DevContextOptions, token: string
       orgId,
     })
 
-    return buildOutput(selectedApp, selectedStore, useCloudflareTunnels, cachedInfo)
+    await enableDeveloperPreview({app: selectedApp, token})
```

Comment:
This seems like a blocking request; my local verbose mode shows about 700ms to execute this. Could we try a fire-and-forget approach to avoid slowing down dev startup time?

Reply:
yesterday I made [that question](https://github.com/Shopify/cli/pull/2171#issuecomment-1597502753) to @ani-shopify :) . We are not enabling it right now so I guess if this automatic action fails we should go on in the same way than previous to this feature.

Reply:
@isaacroldan I think it'll complete quickly enough that there's little need to stop the world over it.

If we want to fail more aggressively, we could put in an `await` later. But I think it's like the cloudflare tunnel, we can continue optimistically at first, rather than waiting nearly a second for this API call to complete.